### PR TITLE
#159068859 Update command to start front end application

### PR DESCRIPTION
**Pull request by anyric**
**What does this PR do?**

This PR updates the start command for the front end application in staging and production environment

**Description of Task to be completed?**
- Use the right command to start the front end application in staging and production

**How should this be manually tested?**

This is handled automatically through the CI/CD pipeline on CircleCi

**Any background context you want to provide?**

N/A

**What are the relevant pivotal tracker stories?**

[#159068859](https://www.pivotaltracker.com/story/show/159068859)

**Screenshots (if appropriate)**

N/A

**Questions:**

N/A

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// Start the server application
+
```
Comment (andela-kakpobome) [inline server.js]: Lose this line

File: server.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+const app = require('./app')
+
+
```
Comment (andela-kakpobome) [inline server.js]: Lose this line

File: server.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+app.listen(3000);
```
Comment (andela-kakpobome) [inline server.js]: Just confirming that, when we hit `activo-staging.andela.com` which is by default on port `80`, you are somehow, mapping that to this port `3000` on your end, right?

Comment (anyric) [reply server.js]: yes

**Pull request merged**